ChangeLog.d/armclang-compile-fix.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix armclang compilation error when targetting certain Arm M-class CPUs
```

Comment:
As far as I can tell, it should be `targeting` (https://english.stackexchange.com/questions/205815/is-targetted-a-standard-british-english-spelling)